runtime/j9vm/jvm.c

```diff
@@ -3589,13 +3589,24 @@ JVM_LoadLibrary(const char* libName)
 	J9JavaVM* javaVM = (J9JavaVM*)BFUjavaVM;
 	J9InternalVMFunctions* vmFuncs = javaVM->internalVMFunctions;
 	J9VMThread* currentThread = vmFuncs->currentVMThread(javaVM);
+	UDATA flagVMAccess = 0;
 
-	Trc_SC_LoadLibrary_Entry(libName);
+	Trc_SC_LoadLibrary_Entry(currentThread, libName);
```

Comment:
I thought we weren't supposed to change tracepoint parameters. Given how infrequently this code will be used, I suggest leaving them as noEnv.

Reply:
A new tracepoint is required if there is a signature change such as the types, order and total number of format specifiers in the Template.
Changes to the Tracepoint Type, Overhead, Level and **NoEnv** parameters, and cosmetic changes to the text in the Template can be made without adding a new tracepoint.

Agreed that this is not a hot path, leaving them as `noEnv`.